src/measure_theory/measure/lebesgue.lean

```diff
@@ -630,3 +630,22 @@ begin
 end
 
 end summable_norm_Icc
+
```

Comment:
Can you add a `/-!` comment here that briefly explains why you state the lemma below with `Iic` on the LHS but `Ioi` on the right, rather than the easier-to-prove version with `Iic` on both sides?

Reply:
OK, done